faq.md

```diff
@@ -3,19 +3,7 @@ layout: flexible
 title: Frequently Asked Questions
 ---
 
-##### Q. Who are the organisers of PyConf Hyderabad 2019 and how can I reach them ?
-PyConf Hyderabad is organised by HydPy, the Hyderabad Python Users Group. You can reach them by dropping an email to [contact@hydpy.org](mailto:contact@hydpy.org) or mention them in their [Twitter handle](https://twitter.com/hydPython).
-
-##### Q. Where can I find updates for PyConf Hyderabad 2019 ?
-All official announcements regarding PyConf Hyderabad 2019 will be made from the [@pyconfhyd](https://twitter.com/pyconfhyd/) Twitter Handle. There is also the [@hydPython](https://twitter.com/hydPython) which is the official Twitter Handle of HydPy where we will also post the updates.
-
-Announcements and updates regarding the Conference is also posted in the [HydPy Mailing list](https://mail.python.org/mailman3/lists/hydpy.python.org/).
-
-##### Q. I want to present a talk/workshop at the conference. What should I do?
-Call for Proposals is now open. Please visit the [CFP page](https://www.papercall.io/pyconf-hyderabad2019) to submit your talk/workshop.
-
-##### Q. I want to volunteer for the conference. What should I do?
-Check out the [Volunteer Page](https://pyconf.hydpy.org/2019/volunteer.html) to know about the roles and responsibilities. Fill the [volunteer form](https://forms.gle/qiKhorSfMjbWCN7Y7) and subscribe to the [HydPy mailing list](https://mail.python.org/mm3/mailman3/lists/hydpy.python.org/) for updates.
-
-##### Q. I am a speaker/volunteer at the conference. Do I need to buy a ticket?
-Yes. At PyConf, everybody buys tickets. Even the event organizers buy tickets. Your ticket helps keep the event affordable for all.
+{% for faqItem in site.data.faq %}
```

Comment:
Since we are using yml files for extracting data now lets put this in a html file and use the usual html markup tags. Also introduce the [Q & A schema](https://schema.org/Question).

Reply:
Q&A schema has lot of attributes and for faq we don't need alot of them so could you provide the attributes that we should include.

Reply:
Also to add, questions and answers are currectly have markdown syntax. To convert the file into HTML, we would have to replace markdown syntax with HTML tags.

Reply:
> Q&A schema has lot of attributes and for faq we don't need alot of them so could you provide the attributes that we should include.

@Tilak999 Just the Question and Answer attribute will be enough

> Also to add, questions and answers are currectly have markdown syntax. To convert the file into HTML, we would have to replace markdown syntax with HTML tags.

Yes replace the .md file with .html file. Since we will pull data from `_data` folder we can update to html now.

Reply:
made the changes, please review.